code/modules/clothing/under/miscellaneous.dm

```diff
@@ -218,6 +218,23 @@
 	worn_state = "nt_skirtsuit"
 	body_parts_covered = UPPER_TORSO|LOWER_TORSO|ARMS
 
+/obj/item/clothing/under/suit_jacket/skirtsuit/color
+	desc = "A black coat with an colour coordinated kerchief accompanied by a swept skirt with a tasteful stripe. Works for every occasion."
+	icon_state = "skirtsuit_recolourable"
+	item_state = "skirtsuit_recolourable"
+	worn_state = "skirtsuit_recolourable"
+	build_from_parts = TRUE
+	worn_overlay = "base"
+	body_parts_covered = UPPER_TORSO|LOWER_TORSO|ARMS
+
+/obj/item/clothing/under/suit_jacket/skirtsuit_command
+	desc = "A black coat with a gold kerchief accompanied by a swept skirt with a respectable golden trim. Works only for Women in Power."
```

Comment:
Ditto with before; I'd either remove or not capitalize 'women' and 'power' shouldn't be capitalized either.

Reply:
Will be adjusted in next update.
![grafik](https://user-images.githubusercontent.com/30761807/153254024-7fecef2a-bda1-4eaf-b0ed-fc0a324d6d19.png)